code/game/objects/items/devices/PDA/cart.dm

```diff
@@ -512,6 +512,23 @@ Code:
 				else
 					menu += "[ldat]"
 
+				menu += "<h4>Located Mops:</h4>"
+
+				ldat = null
+				for (var/obj/item/mop/M in world)
```

Comment:
I'm pretty sure this is an excruciatingly slow way of doing this.

The best solution is to have it so when `/obj/item/mop` (and previously, `/obj/vehicle/ridden/janicart`) `Initialize`, they store themselves in a global list (i.e. `GLOB.mops`, `GLOB.janicarts`) and then remove themselves from that list on `Destroy`. Similar uses of this pattern exist in the form of `GLOB.machines`.